### PR TITLE
refactor: clean-up owners.ts code

### PR DESCRIPTION
**Pull request by jeberhardt**
# Summary | Résumé

There is no issue for this PR.

This started out as a simple task, to change a ternary (`const requestBody = req.body ? req.body : undefined;`
) operator to a nullish operator (`const requestBody = req.body ?? undefined`).

Then I noticed that a little bit of destructuring variables could happen, to further clean up and simplify the code. I also added some tests to ensure that `isValidGovEmail` would handle `null` and `undefined` (because I removed a check for the email within `owners.ts`.

# Test instructions | Instructions pour tester la modification

Load a form and add an email to the Form Access.

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [x] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [x] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: pages/api/id/[form]/owners.ts [hunk 1]
```diff
@@ -80,13 +80,12 @@ export async function activateOrDeactivateFormOwners(
 ): Promise<void> {
   try {
     //Extracting req body
-    const requestBody = req.body ? req.body : undefined;
+    const { email, active } = req.body;
```
Comment (falila) [inline pages/api/id/[form]/owners.ts:83]: This line will throw an error if `req.body` was undefined. To prevent this behaviour it's safe to set a fallback value for `body`.   `const { email, active } = req.body || {}`.

**Review by falila — CHANGES_REQUESTED** [review]

**Review by craigzour — APPROVED** [review]

**Review by falila — APPROVED** [review]
🌮
